_layouts/base.html

```diff
@@ -7,7 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
   <link rel="alternate" type="application/rss+xml" title="Debezium Blog" href="/blog.atom">
-  <link rel="shortcut icon" type="image/png" href="">
+  <link rel="shortcut icon" type="image/png" href="{{ "/favicon.ico" | prepend: site.baseurl }}" >
```

Comment:
Do we have the actual favicon file itself already?

Reply:
> Do we have the actual favicon file itself already?

I add it in this pr.

Reply:
Doh, missed that, thanks. I was put off by the "+1 -1" marker here on GitHub :)